src/python/dxpy/bindings/dxjob.py

```diff
@@ -38,14 +38,16 @@
 from ..utils.local_exec_utils import queue_entry_point
 from ..compat import basestring
 
+log = logging.getLogger(__name__)
+
 #########
 # DXJob #
 #########
 
-def new_dxjob(fn_input, fn_name, name=None, tags=None, properties=None, details=None,
-              instance_type=None, depends_on=None,
-              cluster_spec=None, fpga_driver=None, system_requirements=None, system_requirements_by_executable=None,
-              **kwargs):
+
+def new_dxjob(fn_input, fn_name, name=None, tags=None, properties=None, details=None, instance_type=None,
```

Comment:
manually tested using this applet:
https://github.com/dnanexus/cells/pull/1207/files
- which installs dx-toolkit from this branch and call new_dxjob function inside the applet

Job id (default R470 nvidia driver): https://staging.dnanexus.com/panx/projects/GqbGJV80Xqy5Qzz7B9pqFgyQ/monitor/job/GqzvZFQ0XqyKjYQ2ffYQkkg4
Child job id (expected R535 nvidia driver chnaged during the runtime of the child job creation):
https://staging.dnanexus.com/panx/projects/GqbGJV80Xqy5Qzz7B9pqFgyQ/monitor/job/GqzvbX00XqyG3G980QbyZFB9/try/0